src/SUMMARY.md

```diff
@@ -2,63 +2,63 @@
 
 - [Introduction](README.md)
 - [Overview](overview.md)
-- [Service API coverage](service_api_coverage.md)
-- [Clients API coverage](clients_api_coverage.md)
-- [Parsec Threat Model](threat_model/threat_model.md)
-   - [Recommendations for Secure Deployment](threat_model/secure_deployment.md)
-- [Parsec Rust Client Threat Model](threat_model/rust_client_threat_model.md)
 - [Parsec for users](parsec_users.md)
 - [Parsec for client developers](parsec_client/README.md)
+   - [API Overview](parsec_client/api_overview.md)
+   - [Wire Protocol](parsec_client/wire_protocol.md)
+   - [Status Codes](parsec_client/status_codes.md)
+   - [Writing a new Parsec Client Library](parsec_client/writing_library.md)
    - [Operations](parsec_client/operations/README.md)
-      - [AddClient](parsec_client/operations/add_client.md)
-      - [ListOpcodes](parsec_client/operations/list_opcodes.md)
-      - [ListProviders](parsec_client/operations/list_providers.md)
-      - [ListAuthenticators](parsec_client/operations/list_authenticators.md)
-      - [Ping](parsec_client/operations/ping.md)
-      - [ProveClient](parsec_client/operations/prove_client.md)
-      - [PsaDestroyKey](parsec_client/operations/psa_destroy_key.md)
-      - [PsaExportPublicKey](parsec_client/operations/psa_export_public_key.md)
-      - [PsaExportKey](parsec_client/operations/psa_export_key.md)
-      - [PsaGenerateKey](parsec_client/operations/psa_generate_key.md)
-      - [PsaImportKey](parsec_client/operations/psa_import_key.md)
-      - [PsaRawKeyAgreement](parsec_client/operations/psa_raw_key_agreement.md)
+      - [Parsec Operations Coverage](parsec_client/operations/service_api_coverage.md)
```

Comment:
Moved that here because I wanted to not have too many "top-level" sections. I was not sure if that was part of the client or service documentation. I think it makes sense for it to be in the client section so that the developpers know what they can start implementating and making it work.